src/IncrementalChecker.ts

```diff
@@ -42,7 +48,7 @@ export class IncrementalChecker implements IncrementalCheckerInterface {
     eslints: []
   }));
 
-  private linter?: Linter;
+  private linter?: tslint.Linter;
```

Comment:
Maybe this should be renamed `tslinter` to be specific

Reply:
I would keep this change for later refactoring - there are a lot more places like this and I don't want this PR to grow too much :)